src/pymovements/dataset/dataset_definition.py

```diff
@@ -133,6 +140,7 @@ class DatasetDefinition:
 
     trial_columns: list[str] | None = None
     time_column: str | None = None
+    time_unit: str | None = None
```

Comment:
I think it makes sense to have 'ms' as default.
Most of the datasets i encountered have ms timestamps so it would reduce a bit of boilerplate there